deploy/travis/before_script

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euf -o pipefail
```

Comment:
I'm gonna bet that somebody's been reading https://sipb.mit.edu/doc/safe-shell/...